src/Hal/Report/Html/Reporter.php

```diff
@@ -205,4 +268,12 @@ protected function getTrend($type, $key, $lowIsBetter = false, $highIsBetter = f
             $svg[$r]
         );
     }
+
+    /**
+     * @return bool
+     */
+    private function isHomePage()
+    {
+        return !$this->currentGroup;
```

Comment:
As `currentGroup` is a string, it should be something like this:
 ```suggestion
        return '' === $this->currentGroup;
```